lib/erb_lint/cli.rb

```diff
@@ -215,6 +216,10 @@ def load_config
       if File.exist?(config_filename)
         config = RunnerConfig.new(file_loader.yaml(config_filename), file_loader)
         @config = RunnerConfig.default_for(config)
+      elsif File.exist?(DEPRECATED_CONFIG_FILENAME)
```

Comment:
Can we add a test to check that it'll still work with the deprecated config file but it'll raise a warning?

Reply:
Got it working!